src/utils.jl

```diff
@@ -92,15 +92,22 @@ end
 compute_offset(ŷ::AbstractVector{<:Distributions.UnivariateDistribution}) = expected_value(ŷ)
 compute_offset(ŷ::AbstractVector{<:Real}) = expected_value(ŷ)
 
-function compute_covariate(jointT, W, G, indicator_fns; threshold=0.005)
-    # Compute the indicator values
-    indic_vals = TMLE.indicator_values(indicator_fns, jointT)
-    # Compute density and truncate
+function balancing_weights(G::Machine, W, jointT; threshold=0.005)
     ŷ = MLJBase.predict(G, W)
     d = pdf.(ŷ, jointT)
     plateau!(d, threshold)
-    indic_vals ./= d
-    return indic_vals
+    return 1. ./ d
+end
+
+function covariate_and_weights(jointT, W, G, indicator_fns; threshold=0.005, weighted_fluctuation=false)
```

Comment:
Probably better to call this function "clever_covariate_and_weights"

Reply:
Not for the weighted version, but for the usual fluctuation, could you change the default threshold = 0.005 to 5/(sqrt(n)*ln(n)) where n = data size (from the Gruber 2022 paper https://doi.org/10.1093/aje/kwac087) citing the doi as a comment? (it is unclear what the corresponding formula will be for the weighted version of course).

Reply:
I think I will convert that to a new issue and address it later because it will be breaking and must be dealt with with care.